manifests/g/GuinpinSoft/MakeMKV/v1.17.3/GuinpinSoft.MakeMKV.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Automatically updated by the winget bot at 2023/Feb/03
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: GuinpinSoft.MakeMKV
+PackageVersion: v1.17.3
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- silent
+- interactive
+Installers:
+- Architecture: x64
+  InstallerUrl: https://www.makemkv.com/download/Setup_MakeMKV_v1.17.2.exe

```

Comment:
Appears to still link the old 1.17.2 exe but has the correct updated hash

Reply:
This is because the 1.17.2 URL redirects to the 1.17.3 URL. All of the previous URLs do, in fact. I'd say this is a non-issue and can be fixed in a separate PR

```
PS D:\Git\winget-pkgs> Get-UrlResponse 'https://www.makemkv.com/download/Setup_MakeMKV_v1.6.0.exe'

Name                           Value
----                           -----
ResponseCode                   200
ResponseUrl                    https://www.makemkv.com/download/Setup_MakeMKV_v1.17.3.exe
Headers                        {alt-svc, Server, Cache-Control, Content-Length...}
Response                       https://www.makemkv.com/download/Setup_MakeMKV_v1.17.3.exe
Url                            https://www.makemkv.com/download/Setup_MakeMKV_v1.6.0.exe
ContentLength                  15297637
```

Reply:
@parkerreno your review request is preventing this PR from being able to merge. If you want to make a "suggest" I'll review, or you can mark the conversation as resolved.